src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertPricingUnits.sql

```diff
@@ -60,7 +60,6 @@ VALUES
     ('e6946b09-28a8-4fb5-af57-12ad9247f850', 'callOff'             , 'Call-offs'                    , 'per Call-off'),
     ('720f2d4d-448d-4899-ad40-979b30911ca6', 'carePlan'            , 'care plan/Call-offs'          , 'per custom care plan/Call-off'),
     ('05281ffc-1077-41d5-a253-3077540ef2e9', 'organisation'        , 'organisations'                , 'per organisation'),
-    ('f845fcaa-96ae-4884-aeb6-56576699bfcd', 'nonGP'               , 'non-GP practice organisations', 'per organisation (non-GP practice organisations who consume the care plan service)'),
```

Comment:
With this being removed, is there a referenced ID to be changed (i.e. any catalogue item that uses this pricing unit on the order form)?

Reply:
You are correct, sir. I've covered this in PreDeployment.sql.

The way the DACPAC works probably means we'll have to do to automated deployments – the first will fail because I think DacFx build it's pre-deployment model before the pre-deployment script has run. The second deployment should succeed though. I'm not sure what @gavinwakeley1 plan is for the release but I suspect we may make this change manually first anyway...